int8 CONNECTION_ESTABLISHED=1
int8 LOAD_REQUEST_RECEIVED=2
int8 TRANSFER_REQUEST_RECEIVED=3
int8 START_TRANSFER=4
int8 CARRIER_DETECTED=5
int8 AGV_COMMUNICATION_COMPLETE=6
int8 LOAD_SEQUENCE_INITIATED=7
int8 ERROR=10

int8 state
